config/autoload.php

```diff
@@ -0,0 +1,17 @@
+<?php
+spl_autoload_register('myAutoLoader');
+
+function myAutoLoader($class){
+        if(file_exists("models/$class.php"))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function file_exists() is discouraged](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)